tailwind.config.js

```diff
@@ -1,7 +1,13 @@
+const colors = require('tailwindcss/colors');
+
 module.exports = {
   content: ['./src/**/*.{js,jsx,ts,tsx}', './stories/*'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        THEME: colors.blue,
```

Comment:
is this means its default theme is blue?

Reply:
Yeah. In storybook and if you import the CSS instead of including it into the purge, blue will be the theme.